pom.xml

```diff
@@ -30,7 +29,15 @@
         </developer>
     </developers>
 
+    <scm>
+        <connection>scm:git:git://github.com/jenkinsci/${project.artifactId}-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:jenkinsci/${project.artifactId}-plugin.git</developerConnection>
+        <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
+        <tag>HEAD</tag>
+    </scm>
+
     <properties>
+        <java.level>7</java.level>
```

Comment:
I build this on JDK8.  Any particular reason we want this to result in older JDK7 bytecode?

Reply:
I am very happy to change this to `8` (as I already did for other plugins).
`7` was the default version used with the previous version of the parent pom.

Reply:
I was also suggested by someone that when using Java 8 the Jenkins version should be at least set to 2.60.1 (see https://jenkins.io/changelog-stable/#v2.60.1) as this is the first Jenkins version requiring Java 8.

Is it ok for you to also upgrade this property?

Reply:
I am okay with updating this property to require Java 8.  I don’t think we’re using any Java 8 specific features but in general, the plugins in which this depends requires it so makes sense to me to be consistent.

Thanks for taking the time to update the parent pom.  I’ve done this for a few projects and it is not always easy.